src/components/home/Twotiles.astro

```diff
@@ -6,13 +6,13 @@
     <div class="next">
       <div class="rj">
         <h3>
-          <span class="only-hash">#</span><span class="hashtag">RJ24</span>
+          <span class="only-hash">#</span><span class="hashtag">RJ245</span>
```

Comment:
Hi, Shouldn't it be "RJ25" instead of "RJ245" ?

Reply:
Hey, indeed ! Thanks for reporting, it is fixed !